### PR TITLE
UI Redesign

### PR DESCRIPTION
**Pull request by SeaDve**
It is still not perfect, but most of the features now work.

#### Not working yet:
* Pause and continue buttons
* Label of where to save the video
* Hide some parts of the app when on Wayland (This is already possible but won't add it until other bugs are gone)
* Capture mode buttons
* _delay_button
* Record computer sounds
* Open Folder button in notifications

#### Needs work:
* video_folder_button should only show the folder not the entire directory
* Fix the icons on about dialog and capture mode button
* Code cleaning

#### Visual bugs:
* Adwaita white mode isn't working well (ListBox borders)
* The popover's corner in preferences are cut off

Screenshot from 2021-01-30 19-31-13
Screenshot from 2021-01-30 19-31-21
Screenshot from 2021-01-30 19-31-24
Screenshot from 2021-01-30 19-31-28

### CODE REVIEW

**Comment by amikha1lov** [comment]
it's looks nice, but we should follow GNOME HIG in preferences window

File: src/window.ui [hunk 1]
```diff
@@ -259,464 +186,939 @@
...
+              <object class="GtkStack" id="_record_stop_record_button_stack">
                 <property name="visible">True</property>
-                <property name="can_focus">False</property>
-                <property name="label" translatable="yes">The file will be saved in </property>
-                <property name="justify">center</property>
+                <property name="can-focus">False</property>
+                <property name="transition-duration">150</property>
+                <property name="transition-type">crossfade</property>
+                <child>
+                  <object class="GtkButton" id="_record_button">
```
Comment (amikha1lov) [inline src/window.ui:971]: https://gitlab.com/Latesil/folder-cleaner/-/blob/dev/foldercleaner/user_folder.ui#L29 
<style>
    <class name="suggested-action"/>
</style>

**Review by amikha1lov — CHANGES_REQUESTED** [review]

File: src/window.py [hunk 2]
```diff
@@ -61,23 +61,107 @@ class RecappWindow(Gtk.ApplicationWindow):
...
+        style_context.add_provider_for_screen(
+            screen, provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION
+        )
+        css = b"""
+        decoration , dialog, window.background {
+            border-bottom-left-radius: 8px;
+            border-bottom-right-radius: 8px;
+        }
+
+        #_record_button {
```
[the remark this answers is not in the document]
Comment (SeaDve) [reply src/window.py:114]: fixed it. TIL. Is there some documentation about these style classes?

**Comment by SeaDve** [comment]
Oh shoot, I messed it up. I put it in the wrong folder.

**Comment by SeaDve** [comment]
Ughh, sorry! I think I'll just make a new PR.

**Pull request closed without merging**